clang/include/clang/Driver/Options.td

```diff
@@ -4267,15 +4267,12 @@ defm whole_file : BooleanFFlag<"whole-file">, Group<gfortran_Group>;
 def reuse_exe_EQ : Joined<["-"], "reuse-exe=">, Flags<[CoreOption]>,
   HelpText<"Speed up FPGA aoc compile if the device code in <exe> is unchanged.">,
   MetaVarName<"<exe>">;
-defm sycl : BoolOption<"f", "sycl",
-  LangOpts<"SYCL">, DefaultFalse,
-  PosFlag<SetTrue, [CC1Option], "Enable">, NegFlag<SetFalse, [], "Disable">,
-  BothFlags<[CoreOption], " SYCL kernels compilation for device">>,
-  Group<sycl_Group>;
+def fsycl : Flag<["-"], "fsycl">, Flags<[NoXarchOption]>, Group<sycl_Group>,
```

Comment:
```suggestion
def fsycl : Flag<["-"], "fsycl">, Flags<[NoXarchOption, CoreOption]>, Group<sycl_Group>,
```
Needs to be a core option so it is available on Windows.  Also, should retain the negative variant if possible.

Reply:
Out of curiosity, why retain the negative variant?

Reply:
I see it more from a usability standpoint.  We have provided `-fno-sycl` to users so taking it away may have some impact.

Reply:
Okie dokie, I'll look into it.